src/libraries/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -59,6 +59,8 @@
     <Compile Include="$(CommonPath)System\Net\DebugSafeHandleMinusOneIsInvalid.cs"
              Link="Common\System\Net\DebugSafeHandleMinusOneIsInvalid.cs" />
     <!-- System.Net common -->
+    <Compile Include="$(CommonPath)System\Net\IPEndPointExtensions.cs"
```

Comment:
I was expecting to see deletions from this file, namely everything to do with SocketAddress.

Reply:
There is also System\Net\Internals\IPEndPointExtensions.cs ... and that will go once the cleanup is completed. 
There were some helper functions originally suggested in #86872. But it did not get approved so we will have internal helper function to avoid the whole Serialization/Creation we do for strange socket type. 
If that ever become public API this will go as well.